app.js

```diff
@@ -0,0 +1,59 @@
+const text = document.getElementById("text");
+const btn = document.getElementById("btn");
+const genBG = document.querySelector(".left");
+const colorBox1 = document.getElementById("color-1");
```

Comment:
getElementById yerine querySelectorAll kullanılarak hepsini seçebilirsin. Bu NodeList oluşturduğu için tek tek yazmak yerine forEach methodunu kullanarak hepsine Event Listener verebilirsin.